_posts/2017-08-13-gsoc-karl-1.md

```diff
@@ -0,0 +1,183 @@
+---
+title: "Contributing to Babel: Three GitHub Issues at a Time"
```

Comment:
I wasn't sure about the title since this isn't really about 3 github issues but 3 lessons overall, and mostly non-technical: first is about 2 people saying they were working on it and someone making the PR and some not saying I'm working on this/stopping work or asking if someone is still working on it, second is that snapshot tests have issues (and you can go into more detail about this), and third is timezones and that gathering people together is difficult